src/ensembl/src/content/app/browser/track-panel/track-panel-modal/modal-views/TrackPanelBookmarks.tsx

```diff
@@ -85,6 +77,18 @@ export const PreviouslyViewedLinks = () => {
   );
 };
 
+const buildLabelElement = (labelText: string | string[]) => {
+  if (Array.isArray(labelText)) {
+    return labelText.map((fragment, index) => (
+      <span key={index} className={styles.label}>
```

Comment:
Is `index` a good value for the key?

Reply:
It's not the best (the best is something that guarantees uniqueness), but it's acceptable. From [React docs](https://reactjs.org/docs/lists-and-keys.html):

<img src="https://user-images.githubusercontent.com/6834224/125708482-13ac0fda-7b66-40f0-bdf0-0e5b0603b837.png" width="700px" />

Reply:
Let's leave it as it is then, since it will have just around two items in the array.